### DISCUSSION TITLE
大家最近要好好保重

### DISCUSSION THREAD

**Discussion by Fion-Lin** [1101-GS4719-C/Python-Community-GS4719-C#1792] [General]
最近感冒了，一直咳嗽，也有感受到身邊真的有蠻多人感冒的。這幾天才聽說透過黴漿菌感染的肺炎，雖然我很確定我不是，因為我只是單純抵抗力不好，又單穿一件帽踢半夜騎機車回學校，但還是希望大家要好好注意自己的身體健康，不要感冒! 還聽說最近有新的covid-19病毒株??大家要健康的度過這個聖誕節喔!

**Comment by v0ouo** [comment]
確實，最近溫度變化真的很大，一不小心就會感冒，我大概兩個禮拜前也突然感到喉嚨不舒服，過了一陣子後好不容易不會不舒服又變成不停咳嗽加上卡痰聲音沙啞了，最近狀況才真的有比較好轉，真的要時常注意自己的身體狀況，如果不小心在考試前大感冒的話就糟糕了!

**Comment by 111426027** [comment]
今年冬天天氣真的十分異常，忽冷忽熱，所以平時提升自己的預防力很重要。

**Comment by mrweixd** [comment]
最近感冒真的讓我有些不適，咳嗽一直不停。身邊也有很多人感冒，聽到黴漿菌感染的肺炎還挺擔心的，雖然我確信自己只是抵抗力差，但也提醒大家要好好保重，避免生病。聽說還有新的covid-19病毒株，希望大家能健康度過這個聖誕節，注意身體健康。

**Comment by lyykey** [comment]
不論何時，感冒都是影響人類生活的一大難題。就算現在武漢已經趨緩，還是有很多人飽受病痛所苦，在這裡呼籲大家多多喝水，多睡覺，有良好的生活飲食，不要抽菸喝酒，這樣才能有個健康的身體。大家不論何時，都要好好保重。
祝 平安 順心

**Comment by benbebobo149** [comment]
最近開始換季，真的超煩的，天氣時冷時熱的，一下子穿多了就熱，一下子少穿點就冷。這種時候最容易感冒了，我上個禮拜就是，沒留意被冷到，結果整個週末都在床上打滾。還有最近流感超嚴重的，聽說好多同學都被嚇到了。所以我打算戴口罩，不然就太冒險了。聽說打疫苗全額補助，這可是個好機會啊，我得趕快安排去打。感謝同學的提醒，真是貼心。

**Comment by asd99263fg** [comment]
避免感冒的方法包括保持良好的個人衛生習慣，如經常用肥皂和水洗手，避免觸摸臉部，特別是眼、鼻和口；保持均衡飲食，充足睡眠和適度運動，以增強免疫系統；避免與感冒患者密切接觸，並在公共場所戴口罩；保持室內空氣流通，並定期清潔家居和工作環境；此外，適時接種流感疫苗也有助於預防某些病毒引起的感冒。

**Comment by Tchang323** [comment]
快撐過期末了，大家也要好好保重!XD
